mlbench_core/utils/pytorch/topology.py

```diff
@@ -2,23 +2,33 @@
 import torch
 import torch.distributed as dist
 
+def get_backend_tensor(tensor):
+    if dist.get_backend() == dist.Backend.NCCL:
```

Comment:
Maybe add another condition `dist.is_initialized()` ? The build fails because of that